apollo-backend/src/main/java/io/logz/apollo/blockers/types/BranchBlocker.java

```diff
@@ -32,33 +33,42 @@ public boolean shouldBlock(BlockerInjectableCommons blockerInjectableCommons, De
         DeployableVersion deployableVersion = blockerInjectableCommons.getDeployableVersionDao()
                                                                       .getDeployableVersion(deployment.getDeployableVersionId());
 
+        Boolean shouldBlock = true;
         String repoName = GithubConnector.getRepoNameFromRepositoryUrl(deployableVersion.getGithubRepositoryUrl());
 
-        if (!blockerInjectableCommons.getGithubConnector().isCommitInBranchHistory(repoName,
-                        branchBlockerConfiguration.getBranchName(), deployableVersion.getGitCommitSha())) {
+        for (String branch : branchBlockerConfiguration.getBranchesNames().split(",")) {
```

Comment:
Can you stream that? :)
We got rid of all of those in Apollo, dont want to start again